src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -387,9 +404,14 @@ void PolicyManagerImpl::ProcessAppPolicyCheckResults(
     const policy_table::ApplicationPolicies& app_policies) {
   CheckAppPolicyResults::const_iterator it_results = results.begin();
 
+  typedef std::map<std::string, ActionsForApplicationPolicies>
+      ActionsForApplications;
+  ActionsForApplications actions_for_apps;
+
   for (; results.end() != it_results; ++it_results) {
+    const std::string app_id = it_results->first;
```

Comment:
@AKalinich-Luxoft ref

Reply:
@LuxoftAKutsan was removed after decomposition and moving this staff to separate function. See https://github.com/smartdevicelink/sdl_core/pull/1779/commits/bfdcdd0f21dc83ffb8a607060d3392cad10406a4